modules/react/tabs/lib/TabsItem.tsx

```diff
@@ -81,77 +81,91 @@ export interface TabsItemProps
   tabIndex?: number;
 }
 
-export const StyledTabItem = styled(Box.as('button'))<StyledType & Pick<TabsItemProps, 'gap'>>(
-  ({theme}) => ({
-    ...type.levels.subtext.large,
-    fontWeight: type.properties.fontWeights.medium,
+const tabItemStencil = createStencil({
+  base: {
+    ...system.type.subtext.large,
+    fontFamily: `${system.fontFamily.default}, Arial`,
```

Comment:
I would add the fallbacks that existing previously

Reply:
Will do!